src/conn/conn_log.c

```diff
@@ -277,6 +277,15 @@ __logmgr_config(
 	if (!reconfig) {
 		WT_RET(__wt_config_gets(session, cfg, "log.file_max", &cval));
 		conn->log_file_max = (wt_off_t)cval.val;
+		/*
+		 * With the default log file extend configuration or log file
+		 * extension size larger than configured maximum log file size,
+		 * set the log file extension size to configured maximum log
```

Comment:
minor nit: "...to the configured maximum log..."

Reply:
Done.